golang.go

```diff
@@ -0,0 +1,19 @@
+// package docummentation
+package main
+
+import "context"
+
+// here is a line coment
+
+// FileGetter is used to retriv the code.
```

Comment:
Typo inside comment in word `retriv`. Possible corrections: retrieve (0), retain (0), metric (0),